tools/serverpod_cli/lib/src/analyzer/models/model_parser/model_parser.dart

```diff
@@ -362,19 +366,25 @@ class ModelParser {
 
   static ModelFieldScopeDefinition _parseClassFieldScope(
     YamlMap documentContents,
+    bool serverOnlyClass,
   ) {
     var database = _parseBooleanKey(documentContents, Keyword.database);
     if (database) return ModelFieldScopeDefinition.serverOnly;
```

Comment:
Comment: Not for this PR, but this should be removed on main as it is the deprecated behaviour.